ntbs-service/Models/Enums/AlertType.cs

```diff
@@ -32,6 +32,8 @@ public enum AlertType
         DataQualityDotVotAlert,
         [Display(Name = "Potential duplicate notification")]
         DataQualityPotientialDuplicate,
+        [Display(Name = "ECM value not recorded for child")]

```

Comment:
Shouldn't this be "ECM level" rather than "ECM value"?

Reply:
Will ask Nancy

Reply:
She has updated the alerts page, yes it should be level